examples/ocean-heat-content/ocean-heat-content-1972-03-2000m/ocean-heat-content-1972-03-2000m.json

```diff
@@ -59,6 +73,15 @@
       "href": "./heat_content_anomaly_0-2000_monthly_1972-03.tif",
       "type": "image/tiff; application=geotiff; profile=cloud-optimized",
       "title": "Ocean Heat Content anomalies from WOA09 : heat_content_anomaly 0-2000 m monthly 1.00 degree 1972-03",
+      "created": "2022-04-18T00:00:00Z",
+      "updated": "2022-04-18T00:00:00Z",
```

Comment:
Do we intend to track update datetimes? Is this part of the versioning?

Based on how the NetCDF data is updated, will these always be identical?